Makefile

```diff
@@ -69,36 +69,45 @@ postsubmit-build: setup
 		--artifact-bucket=$(ARTIFACT_BUCKET) \
 		--dry-run=false \
 		--rebuild-all=${REBUILD_ALL}
-	
-.PHONY: kops-prow-arm
-kops-prow-arm: export NODE_INSTANCE_TYPE=t4g.medium
-kops-prow-arm: export NODE_ARCHITECTURE=arm64
-kops-prow-arm: export UBUNTU_RELEASE=focal-20.04
-kops-prow-arm: export UBUNTU_RELEASE_DATE=20221018
-kops-prow-arm: kops-prereqs
+
+.PHONY: kops
+kops: $(if $(CODEBUILD_BUILD_ID),kops-codebuild,kops-prow)
+
+.PHONY: kops-codebuild
+kops-codebuild: KOPS_ENTRYPOINT=development/kops/codebuild.sh
+kops-codebuild: kops-amd kops-arm
+	@echo 'Done kops-codebuild'
+
+.PHONY: kops-prow
+kops-prow: KOPS_ENTRYPOINT=development/kops/prow.sh
+kops-prow: kops-amd kops-arm kops-arm-ubuntu-22
+	@echo 'Done kops-prow'
+
+.PHONY: kops-amd
+kops-amd: export UBUNTU_RELEASE=focal-20.04
+kops-amd: export UBUNTU_RELEASE_DATE=20221018
+kops-amd: kops-prereqs
+	RELEASE=$(RELEASE) $(KOPS_ENTRYPOINT)
+
+.PHONY: kops-arm
+kops-arm: export NODE_INSTANCE_TYPE=t4g.medium
+kops-arm: export NODE_ARCHITECTURE=arm64
+kops-arm: export UBUNTU_RELEASE=focal-20.04
+kops-arm: export UBUNTU_RELEASE_DATE=20221018
```

Comment:
It seems like we have some repeated exported values (like `focal-20.04` and `20221018`). Are these always going to be the same for amd and arm. If so, should be use variables we set at the top?

Reply:
ok, de-duped it by setting the var at the top level of the `kops` target, so it'll default to those releases, and then we can over-ride it in sub-targets as needed